app/models/participant_profile/npq.rb

```diff
@@ -6,10 +6,21 @@ class ParticipantProfile::NPQ < ParticipantProfile
 
   has_one :validation_data, class_name: "NPQValidationData", foreign_key: :id, dependent: :destroy
 
+  self.validation_steps = %i[identity decision].freeze
+
   def npq?
     true
   end
 
+  def approved?
+    validation_decision(:decision).approved?
+  end
+
+  def rejected?
+    decision = validation_decision(:decision)
+    decision.persisted? && !decision.approved?
```

Comment:
This is nullable, right? This treats "no decision yet" as rejection, surely?

Reply:
"no decision yet" mean that the decision is not yet persisted :)

Reply:
If I wanted to leave a comment without selecting a decision - then persist it - this would show "rejected"?

Reply:
For now, you cannot leave a comment without making decision